program/tests/test_create_mint.rs

```diff
@@ -426,18 +739,28 @@ fn test_successful_token_2022_to_spl_token() {
     let resulting_wrapped_mint_account = &result.resulting_accounts[0].1;
     assert_eq!(resulting_wrapped_mint_account.owner, spl_token::id());
 
-    let wrapped_mint_data = Mint::unpack(&resulting_wrapped_mint_account.data).unwrap();
+    let wrapped_mint_data =
+        PodStateWithExtensions::<PodMint>::unpack(&resulting_wrapped_mint_account.data)
+            .unwrap()
+            .base;
+
     assert_eq!(wrapped_mint_data.decimals, MINT_DECIMALS);
     let expected_mint_authority = get_wrapped_mint_authority(&wrapped_mint_address);
     assert_eq!(
-        wrapped_mint_data.mint_authority.unwrap(),
+        wrapped_mint_data
+            .mint_authority
+            .ok_or(ProgramError::InvalidAccountData)
+            .unwrap(),
         expected_mint_authority,
     );
-    assert_eq!(wrapped_mint_data.supply, 0);
-    assert!(wrapped_mint_data.is_initialized);
+    assert_eq!(wrapped_mint_data.supply, PodU64::from(0));
+    assert_eq!(wrapped_mint_data.is_initialized, PodBool::from_bool(true));
     assert_eq!(
-        wrapped_mint_data.freeze_authority.unwrap(),
-        Pubkey::from_str_const(FREEZE_AUTHORITY)
+        wrapped_mint_data
+            .freeze_authority
+            .ok_or(ProgramError::InvalidAccountData)
```

Comment:
nit: same here, is this needed, or is it possible to just unwrap?

Reply:
Same as above with PodCOption